tests/intern-browserstack.ts

```diff
@@ -0,0 +1,14 @@
+export * from './intern';
```

Comment:
A config file can [extend](https://github.com/theintern/intern/blob/master/docs/configuration.md#extends) another, similar to how `tsconfig.json` files work.